stytch/consumer/api/sessions.py

```diff
@@ -279,12 +280,13 @@ async def authenticate_jwt_async(
     # MANUAL(authenticate_jwt_local)(SERVICE_METHOD)
     # ADDIMPORT: import time
     # ADDIMPORT: from stytch.consumer.models.sessions import Session
+    # ADDIMPORT: from typing import Tuple
     def authenticate_jwt_local(
         self,
         session_jwt: str,
         max_token_age_seconds: Optional[int] = None,
         leeway: int = 0,
-    ) -> Optional[Session]:
+    ) -> Optional[Tuple[Dict[str, Any], Dict[str, Any]]]:
```

Comment:
1. Not a fan of tuples
2. This breaks a public interface; let's create a shared helper